proto/cusf/mainchain/v1/validator.proto

```diff
@@ -233,8 +178,8 @@ message GetSidechainProposalsRequest {
 message GetSidechainProposalsResponse {
   message SidechainProposal {
     uint32 sidechain_number = 1;
-    bytes data = 2;
-    bytes data_hash = 3;
+    google.protobuf.BytesValue data = 2;
```

Comment:
Would it make sense to decode this data into a meaningful message? I.e. `version`, `title`, `description` and the two hashes

https://github.com/bitcoin/bips/blob/master/bip-0300.mediawiki#m1----propose-sidechain